datadog_checks_base/tests/test_similar.py

```diff
@@ -178,3 +178,18 @@ def test__build_similar_elements__service_check_hostname(self, aggregator):
 
         # expect similar metrics in a similarity order
         assert similar_service_checks[0][1].name == 'test.similar3'
+
+    def test__build_similar_elements__histogram_buckets(self, aggregator):
+        check = AgentCheck()
+
+        check.submit_histogram_bucket('histogram.bucket', 1, 10.0, 125.0, True, "hostname", ["tag2"])
+        check.submit_histogram_bucket('histogram.bucket', 1, 125.0, 312.0, True, "hostname", ["tag1"])
+        check.submit_histogram_bucket('histogram.bucket', 1, 0.0, 10.0, True, "hostname", ["tag1"])
+
+        expected_histogram_bucket = HistogramBucketStub('histogram.bucket', 1, 0.0, 10.0, True, "hostname", ["tag1"])
+        similar_histogram_bucket = similar._build_similar_elements(
+            expected_histogram_bucket, aggregator._histogram_buckets
+        )
+
+        # expect similar metrics in a similarity order
+        assert similar_histogram_bucket[0][1].name == 'histogram.bucket'
```

Comment:
This assertion will be probably always true since the metric name is the same for all 3 cases: `histogram.bucket`

Reply:
1. thanks for spotting this 👍 , addressed via https://github.com/DataDog/integrations-core/pull/4419 
2. coverage should already be good as I'm doing several attribute tests in one test instead of having one test per attribute 😉